sevntu-checkstyle-sonar-plugin/src/test/java/com/github/sevntu/checkstyle/sonar/CheckstyleExtensionRulesDefinitionTest.java

```diff
@@ -0,0 +1,56 @@
+////////////////////////////////////////////////////////////////////////////////
+// checkstyle: Checks Java source code for adherence to a set of rules.
+// Copyright (C) 2001-2019 the original author or authors.
+//
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public
+// License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+//
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+// Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+////////////////////////////////////////////////////////////////////////////////
+
+package com.github.sevntu.checkstyle.sonar;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.sonar.api.server.rule.RulesDefinition;
+import org.sonar.api.server.rule.RulesDefinitionXmlLoader;
+
+import java.util.List;
+
+public class CheckstyleExtensionRulesDefinitionTest {
+
+    @Test
+    public void testAllRulesValid() {
+        CheckstyleExtensionRulesDefinition definition = new CheckstyleExtensionRulesDefinition(new RulesDefinitionXmlLoader());
```

Comment:
It looks like checkstyle isn't being run on these sources. Line lengths should be 100 characters max.

Reply:
Yes. **test/java** is not included in maven-checkstyle-plugin configuration. By default only **main/java** is included. I'll take a look if I can fix it in a separate commit of this PR.

Reply:
@rnveach, fixed